include/openenclave/attestation/sgx/eeid_attester.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+// This file provides an implementation of EEID attester and verifier plugins.
+
+#ifndef _OE_EEID_ATTESTER_H
+#define _OE_EEID_ATTESTER_H
+
+#include <openenclave/internal/sgx/eeid_plugin.h>
+
+oe_result_t eeid_get_evidence(
```

Comment:
This looks like it's a new public header.  If so, https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says
"DO use doxygen comments, with [in,out] direction annotation in all public API headers."

Reply:
Done.

Reply:
(Also moved unnecessary declarations out of the header files.)